src/main/java/thePackmaster/powers/goddessofexplosionspack/RedshiftPower.java

```diff
@@ -0,0 +1,54 @@
+package thePackmaster.powers.goddessofexplosionspack;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.TexLoader;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+
+public class RedshiftPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("RedshiftPower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String DESCRIPTIONS[] = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+    private static int idOffset;
+
+    public RedshiftPower(AbstractCreature owner, int amount) {
+        super(POWER_ID+idOffset, NAME, PowerType.BUFF, true, owner, amount);
+
+        // Prevent offset from screwing up the icons.
+        Texture normalTexture = TexLoader.getTexture(SpireAnniversary5Mod.modID + "Resources/images/powers/RedshiftPower32.png");
+        Texture hiDefImage = TexLoader.getTexture(SpireAnniversary5Mod.modID + "Resources/images/powers/RedshiftPower84.png");
+        if (hiDefImage != null) {
+            region128 = new TextureAtlas.AtlasRegion(hiDefImage, 0, 0, hiDefImage.getWidth(), hiDefImage.getHeight());
+            if (normalTexture != null)
+                region48 = new TextureAtlas.AtlasRegion(normalTexture, 0, 0, normalTexture.getWidth(), normalTexture.getHeight());
+        } else if (normalTexture != null) {
+            this.img = normalTexture;
+            region48 = new TextureAtlas.AtlasRegion(normalTexture, 0, 0, normalTexture.getWidth(), normalTexture.getHeight());
+        }
```

Comment:
https://github.com/kiooeht/StSLib/blob/master/src/main/java/com/evacipated/cardcrawl/mod/stslib/powers/interfaces/NonStackablePower.java
Also allows for conditional stacking so you could stack it when wanted